superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -197,6 +198,9 @@ class HeaderActionsDropdown extends React.PureComponent {
           menu.style.visibility = 'visible';
         });
         this.props.logEvent?.(LOG_ACTIONS_DASHBOARD_DOWNLOAD_AS_IMAGE);
+        this.props.addInfoToast(
+          t('Image download takes a long time, please wait.'),
```

Comment:
I think we can use a generic message to indicate a file is being prepared for download. The message would be the same for image and pdf.
Eg: "Preparing file for download, please wait."